docs/tutorials/integrations/gitlab-ci.md

```diff
@@ -85,8 +85,6 @@ container_scanning:
     FULL_IMAGE_NAME: $CI_REGISTRY_IMAGE:$CI_COMMIT_REF_SLUG
   script:
     - trivy --version
-    # cache cleanup is needed when scanning images with the same tags, it does not remove the database
-    - time trivy image --clear-cache
```

Comment:
I'm not sure I understand why this was necessary.
Would removing this break GitLab integration?

Reply:
The cache is based on image ID and will not be used even when the same tag is used for different images, like the comment says. In other words, the comment is wrong and I believe we don't need `--clear-cache` here.

Reply:
> The cache is based on image ID and will not be used even when the same tag is used for different images, like the comment says

yeah. I know this, but I doubted correctness of comment.
Thanks that clarified this for me.